dist/s_clang_format

```diff
@@ -6,16 +6,29 @@ t=__wt.$$
 trap 'rm -rf $t' 0 1 2 3 13 15
 
 download_clang_format() {
-    if [ `uname` = "Linux" ]; then
-        curl https://s3.amazonaws.com/boxes.10gen.com/build/clang-format-llvm-10.0.0-x86_64-linux-gnu-ubuntu-20.04.tar-1.gz -o dist/clang-format.tar.gz
-        tar --strip=1 -C dist/ -xf dist/clang-format.tar.gz clang-format-llvm-10.0.0-x86_64-linux-gnu-ubuntu-20.04/clang-format && rm dist/clang-format.tar.gz
-    elif [ `uname` = "Darwin" ]; then
-        curl https://s3.amazonaws.com/boxes.10gen.com/build/build/clang-format-llvm-10.0.0-x86_64-apple-darwin.tar.gz -o dist/clang-format.tar.gz
-        tar --strip=1 -C dist/ -xf dist/clang-format.tar.gz clang-format-llvm-10.0.0-x86_64-apple-darwin/clang-format && rm dist/clang-format.tar.gz
+    version=$1
+    # FIXME-WT-10583
+    arch=$(uname -m)
+    if [ ! "$arch" = "x86_64" ]; then
+        echo "$0: unsupported architecture $arch"
+        return 1
+    fi
+
+    archive=./dist/clang-format.tar.gz
```

Comment:
The below curl & tar commands use the variable with `./` preceded. Is the `./` here in variable value necessary?

Reply:
It's not, update here 5af36d4764b48c0291ca3bdf4fb8926dd2065184